src/components/config/parse.ts

```diff
@@ -48,7 +53,11 @@ export const parseConfig = (configPath: string, type: string): Config => {
       return (cfg as unknown) as Config
     }
 
-    return JSON.parse(configString)
+    if (omitNotifications) {
+      return JSON.parse(configString) as Omit<Config, 'notifications'>
+    }
+
+    return JSON.parse(configString) as Config
```

Comment:
This line 57 and 60 actually return the same thing. You just change the types but the values itself in runtime are the same. To prove, see [this sample in this typescript playground](https://www.typescriptlang.org/play?#code/ATBQEsDsBcFMCcBmBDAxrYBhA9pR4BzYAb1BBEmQFtYAuYAZ2nigIG4zzkC7hIBXKgCMEHAL6hQqXE2AATZNGQBlZq2ABeYAANQpAJAAiSjUP1DyQwBpQR7rDPAAjKAm6pM6PMXJNwAFLKAPIAcgB0AA7I8AywABQKSqoskAQAlMDIDFi4+AQekAzYADawYcXYBHHEichiaZLShV61AEx+gaGR0bEJPsmsGVnAQVTg0AA8OHiEVsAA5PbzAHwFRaXlldVt9UA). Press run then check the log.

If you actually want to literally remove the `notifications` property, you need to something like

```
const obj = JSON.parse(configString)
delete obj.notifications
```